src/main/java/org/team1540/robot2019/Tuning.java

```diff
@@ -130,4 +130,11 @@
     public static double climberTolerance = 1000;
 
     public static double axisButtonThreshold = 0.3;
+
+    // auto-lineup
+    public static double drivetrainRadiusMeters = 0.305; // This is for pandora (Seems to work... hmmm) TODO: Change this
```

Comment:
um

Reply:
For some reason pandora’s radius seems to work better...

Reply:
...ok then